src/antsibull/data/ansible-readme.rst

```diff
@@ -56,6 +56,10 @@ Issues with plugins and modules in the Ansible package should be reported
 on the individual collection's issue tracker.
 Issues with ``ansible-core`` should be reported on
 the `ansible-core issue tracker <https://github.com/ansible/ansible/issues/>`_.
+Issues with the ``ansible`` package build process or serious bugs or
+vulnerabilities in a collection that are not addressed after opening an issue
+in the collection's issue tracker should be reported on
+`ansible-build-data's issue tracker <https://github.com/ansible-community/ansible-build-data/issues>`_.
```

Comment:
Should we say at all what kind of issues we accept in ansible-build-data here? Can't we _just_ refer to the README? Something like:

> If you think the above issue trackers are the wrong place for your issue, there is also the `ansible-build-data's issue tracker <https://github.com/ansible-community/ansible-build-data/issues>`_. Please refer to the `ansible-build-data's README <https://github.com/ansible-community/ansible-build-data/blob/main/README.md>` for more details.

This lacks the information about "serious bugs or vulnerabilities in a collection that are not addressed after opening an issue in the collection's issue tracker", though... but I think we should mention the README somehow here. We have a lot more details about acceptable issues there, far more than we could or should go into here.